README.md

```diff
@@ -669,6 +669,29 @@ This means that you can carry on splitting as much as you like.
 const var [[[getScore, setScore], setScore], setScore] = use(0)!
 ```
 
+## Performance Optimization
+
+DreamBerd includes a state-of-the-art optimization system, now accessible using the `vroom` keyword. This special keyword can be spelled with additional o's if more optimization is required.
+
+```java
+fn is_prime (num) => {
+   var var return = true
```

Comment:
I see a new concept that's not described anywhere: assigning to the `return` statement.